src/App.jsx

```diff
@@ -13,7 +14,7 @@ function App() {
       <Navbar/>
       <Routes>
         <Route path="/" element={<Articles/>}/>
-
+        <Route path="/Topics" element={<Topics/>}/>
```

Comment:
Try not to add any additional code that isn't required to complete the task that this branch is referring to - so that we can keep all relevant code on the relevant branch 
So anything to do with topics shouldn't be part of this PR